master/reference/calicoctl/resources/policy.md

```diff
@@ -92,8 +92,10 @@ applied action is deny.
 |-------------|-----------------------------|-------------------|--------|------------|
 | tag (deprecated)      | Match on tag. |  | string | |
 | notTag (deprecated)   | Negative match on tag. |  | string | |
-| net    | Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
-| notNet | Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr | |
+| net                   | Deprecated (use "nets" instead): Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
```

Comment:
Do we have to deprecate the singular forms?  I would guess (before looking at the code changes) that the cost of this is very small, so why not keep both singular and plural forms on equal footing?